src/modules/common/components/PageNotAvailable.js

```diff
@@ -0,0 +1,51 @@
+// @flow
+
+import * as React from 'react'
+import styled from 'styled-components/native'
+import type { ThemeType } from '../../theme/constants/theme'
+import { LanguageModel } from '@integreat-app/integreat-api-client'
+import Selector from './Selector'
+import SelectorItemModel from '../models/SelectorItemModel'
+import { Text } from 'react-native-elements/src/index'
+import Caption from './Caption'
+import { TFunction } from 'react-i18next'
+
+const Wrapper = styled.View`
+  position: absolute;  
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  background-color: ${props => props.theme.colors.backgroundColor};
+  align-items: center;
+`
+
+type PropsType = {
+  theme: ThemeType,
+  city: string,
+  languages: Array<LanguageModel>,
+  changeLanguage: (city: string, language: string) => void,
+  t: TFunction
+}
+
+class PageNotAvailable extends React.Component<PropsType> {
+  onPress = (model: LanguageModel) => {
+    this.props.changeLanguage(this.props.city, model.code)
+  }
+
+  render () {
+    const {t, languages, theme} = this.props
+    return <Wrapper theme={theme}>
+      <Caption title={t('notFoundLanguage')} />
```

Comment:
Kind of conflicting with the component name `PageNotAvailable`. Either this should be `LanguageNotAvailablePage` (or something similar) or the caption should not be constant.

Reply:
Renamed the locale key to languageNotAvailable and the component to LanguageNotAvailablePage.